packages/wonder-blocks-tooltip/util/visibility-modifier.test.js

```diff
@@ -0,0 +1,185 @@
+                    instance: {reference: psuedoAnchorElement},
+                    attributes: {},
+                    hide: false,
+                };
+                // Flow doesn't know this is a jest mock $FlowFixMe
+                Core.getElementIntersection.mockReturnValue({
+                    horizontal: "before",
+                    vertical: "within",
+                });
+
+                // Act
+                const result = visibilityModifierDefaultConfig.fn(data);
+
+                // Assert
+                expect(result.hide).toBeTruthy();
+                expect(result.attributes["x-out-of-boundaries"]).toBe("");
+            });
+
+            test("Obscured vertically only, is not visible", () => {
+                // Arrange
+                const psuedoAnchorElement = {
+                    getBoundingClientRect: () => ({
+                        top: 0,
+                        left: 0,
+                        right: 100,
+                        bottom: 100,
+                    }),
+                };
+                const data = {
+                    instance: {reference: psuedoAnchorElement},
+                    attributes: {
+                        "x-out-of-boundaries": "",
+                    },
+                    hide: true,
+                };
+                // Flow doesn't know this is a jest mock $FlowFixMe
+                Core.getElementIntersection.mockReturnValue({
+                    horizontal: "within",
+                    vertical: "before",
+                });
+
+                // Act
+                const result = visibilityModifierDefaultConfig.fn(data);
+
+                // Assert
+                expect(result.hide).toBeTruthy();
+                expect(result.attributes["x-out-of-boundaries"]).toBe("");
+            });
+
+            test("Obscured totally, is not visible", () => {
+                // Arrange
+                const psuedoAnchorElement = {
+                    getBoundingClientRect: () => ({
+                        top: 0,
+                        left: 0,
+                        right: 100,
+                        bottom: 100,
+                    }),
+                };
+                const data = {
+                    instance: {reference: psuedoAnchorElement},
+                    attributes: {},
+                    hide: false,
+                };
+                // Flow doesn't know this is a jest mock $FlowFixMe
+                Core.getElementIntersection.mockReturnValue({
+                    horizontal: "after",
+                    vertical: "after",
+                });
+
+                // Act
+                const result = visibilityModifierDefaultConfig.fn(data);
+
+                // Assert
+                expect(result.hide).toBeTruthy();
+                expect(result.attributes["x-out-of-boundaries"]).toBe("");
+            });
+        });
+
+        test("Obscured by fixed or absolute components, is not visible", () => {
```

Comment:
I see... we're making the assumption that by having `getElementIntersection` return `CompletelyWithin` and `isObscured` return `true` that we're simulating a similar situation to what would happen if the anchor was obscured by a fixed or absolute components.  Can you add a comment as it's not immediately obvious how the test title follows from the test?